payloads/external/edk2/Kconfig

```diff
@@ -282,6 +282,18 @@ config EDK2_DASHARO_NETWORK_CONFIG
 	help
 	  Enables the EDK2 network stack configuration submenu in Dasharo System Features.
 
+config EDK2_DASHARO_NETWORK_BOOT_DEFAULT_ENABLE
+	bool "Enable EDK2 network boot option by default"
+	default n
+	help
+	  Enables the network boot option in EDK2 by default.
+
+config EDK2_SETUP_PASSWORD
```

Comment:
@3mdeb-karolzmyslowski 

About this commit name:

`.config, Makefile.inc, Kconfig, Makefile: Add setup password `

If you look at the changes, it should go into two separate commits. One of them adds new option to edk2 payload, and is independend of the hardware platform.

The other commit should simply enable this option in certain NovaCustom configs.